example/index.html

```diff
@@ -2,7 +2,8 @@
 <html>
   <head>
     <title>Markdown Demo</title>
-    <link rel="stylesheet" type="text/css" href="https://cdn.readme.io/css/bundle-hub2.css" />
+    <link rel="stylesheet" type="text/css" href="https://cdn.readme.io/public/hub/web/main.3f4ed9ef7e8b221de43c.css" />
+    <link rel="stylesheet" type="text/css" href="https://cdn.readme.io/public/hub/web/3435.9200159a8fec79d971c4.css" />
```

Comment:
@rafegoldberg is there a better way to pull in general app styles?

Reply:
Technically we shouldn't need the `bundle-hub2.css` _or_ the React app styles here, but I don't know a better way of including ‘em in the demo app unfortunately, no…

Reply:
I guess you could move these in to the `demo.scss` as `@import` statements. Still manual, but might _feel_ a little cleaner?